chfunds.permitted-roles.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This constant DRUPAL_ROLE_PERMISSIONS holds the drupal role and the respective with CH Permissions in serialised format
```

Comment:
Copyedit change to 'This constant DRUPAL_ROLE_PERMISSIONS holds an array of drupal roles and permissions that are to be programmatically assigned to each role.'

Reply:
Done. Submitted PR https://github.com/CanadaHelps/chfunds/pull/8